### PR TITLE
Start assignment

### PR DESCRIPTION
**Pull request by ShaunWMusic**

### CODE REVIEW

**Comment by ShaunWMusic** [comment]
- [x] Fork Repository `tiy-tn-fee-2016-fall/10-twitch` on Github
- [x] Run `hub clone 10-twitch` to clone your repo locally
- [x] Run `npm install` in project directory to install test and build dependencies
- [x] Run `hub remote add class tiy-tn-fee-2016-fall/10-twitch` so that you can pull new changes as they come
- [x] Create `develop` branch

**Comment by ShaunWMusic** [comment]
- [x] Work through tests for '01-create-message-item'
- [x] Work through tests for '02-add-message-item'
- [x] Work through tests for '03-sum-from-list'
- [x] Work through tests for '04-reverse-list'

**Comment by ShaunWMusic** [comment]
- [x] Run `git pull --no-edit class master` to bring in updates to the assignment
- [x] Look at the tests in the `tests/twitch` folder
  - [x] Work through tests for '01-create-game-item'
  - [x] Work through tests for '02-add-games-to-list'
  - [ ] Work through tests for '03-search-twitch'
- [x] Run `git pull --no-edit class master` to bring in updates to the assignment
- [ ] Write HTML for the Twitch left sidebar
- [ ] Style the Twitch left sidebar
- [ ] Write HTML for the Twitch right sidebar
- [ ] Style the Twitch right sidebar
- [ ] Style the game results

File: app/index.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
 import 'whatwg-fetch';
+<<<<<<< HEAD
```
Comment (rtablada) [inline app/index.js]: Looks like you committed merge conflicts. Remove this line that says `HEAD` all the way through the `=======`.

File: app/index.js [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+// import './exercise/reverse-list';
+// import './exercise/sum-from-list';
+=======
 // Import `data.json`
 // Import `addGamesToList` from `twitch/add-games-to-list`
 
 
 // Look up a `ul` element with a class called `results`
 // Add games to the results list based on the data from `data.json`
+>>>>>>> eb1ab13731653051706223d3532d706ecdfac751
```
Comment (rtablada) [inline app/index.js]: Then remove this line which is showing the end of changes for things that you merged in from my branch.

**Review by rtablada — CHANGES_REQUESTED** [review]

**Pull request merged**